proj/src/reducers/users/userReducer.js

```diff
@@ -2,13 +2,13 @@ import { handleActions, combineActions } from 'redux-actions';
 import { set } from 'lodash/fp'
 
 const initialState = {
-  currUser: null
+  currentUser: null
 };
 
 export default handleActions({
   SET_LOGGED_IN: (state, { email, userId }) =>
-    set('currUser', {email: email, userId: userId}, state)
+    set('currentUser', {email: email, userId: userId}, state)
```

Comment:
even though this will probably get refactored. you could have used the property shorthand style.
set('currentUser', { email, userId }, state)

BUT, it will even be easier after refactor since all you have now is email and user you can just return { email, userId }